src/components/TableOfContents/index.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from './styles';
 import { useEffect } from 'react';
 import { useRouter } from 'next/router';
-import Feedback from '../Feedback';
+import Feedback from '../FeedbackSticky';
```

Comment:
@katiegoines, I think we might need to move it out of this component because there are some pages that don't have the table of contents, for instance: https://pr-5334.d39agkzf0cwtdz.amplifyapp.com/cli/commands/add/

Maybe it could be moved into the `<ContentStyle>` component inside `/Page/index.tsx`?